app/controllers/tweet_controller.rb

```diff
@@ -0,0 +1,74 @@
+class TweetsController < ApplicationController
+
+
+  get '/tweets' do
+    if logged_in?
+      @tweets = Tweet.all
+      erb :'tweets/tweets'
+    else
+      redirect to "/login"
+    end
+  end
+
+  get '/tweets/new' do 
+    if logged_in?
+      erb :'tweets/create_tweet'
+    else
+      redirect to "/login"
+    end
+  end
+
+  post '/tweets/new' do 
+    if params[:content] == ""
+      redirect to "/tweets/new"
+    else
+      @user = User.find_by_id(session[:id])
+      @tweet = Tweet.create(content: params[:content], user_id: @user.id)
+    end
+  end
+
+  get '/tweets/:id' do 
+    if logged_in?
+      @tweet = Tweet.find_by_id(params[:id])
+      erb :'tweets/show_tweet'
+    else
+      redirect to "/login"
+    end
+  end
+
+  get '/tweets/:id/edit' do 
+    if logged_in?
+      @tweet = Tweet.find_by_id(params[:id])
```

Comment:
Shouldn't you verify a tweet belongs to the `current_user` (and not just that any user is logged in) before allowing them to edit?